enterprise-suite/tests/smoke_port-forward

```diff
@@ -5,6 +5,7 @@
 #set -x
 
 source smokecommon
+if [[ $context == "GKE" ]]; then init_gke_ingress ; fi
```

Comment:
Whatever init gke needs should be done in a separate script, call it `gke_init`, and should be done once before anything else is done, at which point all the tests kick off.

What you've done is ask for init_gke_ingress to be started simultaneously by every test.  no bueno